main.tf

```diff
@@ -102,6 +106,10 @@ resource "aws_ecs_service" "service_background" {
   lifecycle {
     ignore_changes = [task_definition]
   }
+  ordered_placement_strategy {
```

Comment:
Will this also make sure they are in separate availability zones ?

Reply:
We should check node utilization once we do this. Just hoping that in spreading the tasks across instances and availability zones does not mean inefficient utilization.